.eslintrc

```diff
@@ -0,0 +1,37 @@
+{
+    "rules": {
+        "camelcase": ["off", {"properties": "always"}],
```

Comment:
Tbh the camelcase turned off makes me a bit sad 😢 but I guess we should stick with the original config?

Reply:
I feel the same way on both counts.